eosio/deployment/BancorX.abi

```diff
@@ -88,6 +88,14 @@
                 {
                     "name": "max_destroy_limit",
                     "type": "uint64"
+                },
+                {
+                    "name": "rewards_receiver",
+                    "type": "name"
```

Comment:
The type of this argument is "name"? Is this a valid type, or a typo? And if it's a typo, does it still work? It appears in several places.

Reply:
yes this is a type.

Reply:
https://developers.eos.io/manuals/eosio.cdt/v1.8/structeosio_1_1name

Reply:
it will no pass compilation if this is a typo...